lib/OpenQA/WebAPI/Controller/API/V1/Feature.pm

```diff
@@ -0,0 +1,21 @@
+package OpenQA::WebAPI::Controller::API::V1::Feature;
+use strict;
+use warnings;
+use Mojo::Base 'Mojolicious::Controller';
+
+sub informed {
+    my ($self)  = @_;
+    my $user    = $self->current_user;
+    my $version = $self->param('version');
+    $user->update({last_login_version => $version});
+    my $seen = $self->param('seen');
+    $user->update({feature_informed => $seen});
+}
+
+sub check {
+    my ($self) = @_;
+    my $user = $self->current_user;
+    $self->respond_to(json => {json => {version => $user->last_login_version, seen => $user->feature_informed}},);
```

Comment:
this check route implies you want to query the server for every page load. This sounds problematic. You better check while rendering the page - if you need the feature feature at all.